CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## [v1.6.1](https://github.com/auth0/auth0-spa-js/tree/v1.6.1) (2020-01-07)
+
+**Fixed**
+
+Included core-js polyfill for `String.includes` to fix an issue with browser-tabs-lock in IE11 [stevehobbsdev](https://github.com/stevehobbsdev) - https://github.com/auth0/auth0-spa-js/pull/325
```

Comment:
@stevehobbsdev I think the script that generates this is broken. Each line should be be prepended a `-` or similar

Reply:
There is no script to generate this, it is manual. I have an item in the backlog to align it with other SDKs that use webtask.

For now, I've just formatted it the same way that previous entries have been done.